### PR TITLE
docs: correct search over new field docs

### PR DESCRIPTION
**Pull request by shubhamg931**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: docs/debugging.md [hunk 1]
```diff
@@ -191,6 +191,6 @@ docker rm -f $(docker ps -aq)
 ```
 2. Drop all DataHub's docker volumes.
 ```
-docker volume rm -f $(docker volume ls -f name=datahub_*  -q)
+docker volume rm -f $(docker volume ls -f name=datahub_  -q)
```
Comment (shubhamg931) [inline docs/debugging.md:194]: name filter in docker requires only part of the volume name. Therefore, using `name=datahub_*` results in an empty list of volumes and thus the setup does not get nuked properly

File: docs/how/search-over-new-field.md [hunk 2]
```diff
@@ -256,7 +256,7 @@ curl 'http://localhost:8080/corpUsers?action=ingest' -X POST -H 'X-RestLi-Protoc
 Once the ingestion is done, you can test your changes by issuing search queries. Here is an example query with response.
 
 ```sh
-curl "http://localhost:8080/corpUsers?q=search&input=ai200" -H 'X-RestLi-Protocol-Version: 2.0.0' -s | jq
+curl "http://localhost:8080/corpUsers?q=search&input=ai100" -H 'X-RestLi-Protocol-Version: 2.0.0' -s | jq
```
Comment (shubhamg931) [inline docs/how/search-over-new-field.md:259]: The course name being ingested as data during ingestion is `AI100: Introduction to Artificial Intelligence`

Searching `ai200` gives empty results.

File: docs/debugging.md (hunk 1 above)
Comment (mars-lan) [reply docs/debugging.md:194]: Makes sense.

**Review by mars-lan — APPROVED** [review]
Thanks for fixing the doc!

**Pull request merged**